opendatahub-io/data-science-pipelines-operator
# Parameterized HealthCheck Timeouts

## The issue resolved by this Pull Request:
Resolves https://github.com/opendatahub-io/data-science-pipelines-operator/issues/328

## Description of your changes:
Parameterized `DBConnectionTimeout` and `ObjStoreConnectionTimeout`.
`MaxConcurrentReconciles` was already parameterized.

## Testing instructions

Set the variables `DB_CONNECTION_TIMEOUT` and `OBJ_STORE_CONNECTION_TIMEOUT`, run DSPO, create a DSPA, and ensure the DSPA deploys correctly.

## Checklist
